cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+from django.db import connection
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command that waits for database to be available"""
+
+    def handle(self, *args, **options):
+        """Handle the command"""
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        while not db_conn:
+            try:
+                connection.ensure_connection()
+                db_conn = True
+            except OperationalError:
+                self.stdout.write("Database unavailable, waiting 5 second...")
+                time.sleep(5)
```

Comment:
Waiting for 5 seconds could be decreased, reduce sleeping time to 1 second.

Reply:
done